### DISCUSSION TITLE
The evaporation term source

### DISCUSSION THREAD

**Discussion by RachelMeng0096** [AMReX-Combustion/PeleC#796] [Q&A]
I am currently simulating a spray jet flame in crossflow using PeleC, though it might be more accurate to say PeleMP. I am trying to obtain the evaporation term source of a droplet. Despite outputting all variables, I have been unable to locate this particular variable. Could you please offer some guidance? I would greatly appreciate any advice you can provide.

**Comment by baperry2** [comment]
Unfortunately, I am not aware of an existing interface in PeleC for plotting source terms from the spray. They obviously get computed in the simulation, but you may need to make some modifications to the code to output them.
